bundle/manifests/1.4.1/skupper-operator.v1.4.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,297 @@
+                image: quay.io/skupper/site-controller@sha256:2ae8c0b3af3b9dd3c80e777c2cec0139f2bef952343c9549505fd63466fc2b92
+                name: manager
+                securityContext:
+                  allowPrivilegeEscalation: false
+              securityContext:
+                runAsNonRoot: true
+              serviceAccountName: skupper-site-controller
+              terminationGracePeriodSeconds: 10
+      permissions:
+      - rules:
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - statefulsets
+          - daemonsets
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - apps.openshift.io
+          resources:
+          - deploymentconfigs
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ''
+          resources:
+          - pods
+          - pods/exec
+          - secrets
+          - configmaps
+          - serviceaccounts
+          - services
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - rolebindings
+          - roles
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+        - apiGroups:
+          - networking.k8s.io
+          resources:
+          - ingresses
+          - networkpolicies
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+        - apiGroups:
+          - projectcontour.io
+          resources:
+          - httpproxies
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+        serviceAccountName: skupper-site-controller
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - skupper
+  - service
+  - mesh
+  - van
+  links:
+  - name: Skupper Operator
+    url: https://github.com/skupperproject/skupper-operator
+  maintainers:
+  - email: skupper@googlegroups.com
+    name: Skupper Community
+  maturity: alpha
+  provider:
+    name: Skupper Project
+    url: https://skupper.io
+  replaces: skupper-operator.v1.3.0
+  skips:
+  - skupper-operator.v1.4.0-rc2
+  - skupper-operator.v1.4.0-rc2
```

Comment:
is this repeated for a reason? should it skip 1.4.0-rc4 as well?

Reply:
Fixing.